tabpfn_client/client.py

```diff
@@ -135,7 +142,15 @@ def predict(self, train_set_uid: str, x_test, tabpfn_config: dict | None = None)
 
         self._validate_response(response, "predict")
 
-        return np.array(response.json()["y_pred_proba"])
+        result = response.json()[task]
+
```

Comment:
Could you include comment to tell why we handle the result based on the types? i.e. mention we could get classification or regression results. Might be helpful for other developers :)

Reply:
OR

Would it be better if we move this logic to `TabPFNClassifier` and `TabPFNRegressor`?